src/main/java/de/rwth/idsg/steve/config/SecurityConfiguration.java

```diff
@@ -100,23 +97,90 @@ public WebSecurityCustomizer webSecurityCustomizer() {
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         final String prefix = CONFIG.getSpringManagerMapping();
-
         return http
-            .authorizeHttpRequests(
-                req -> req.antMatchers(prefix + "/**").hasRole("ADMIN")
-            )
+            .authorizeRequests(
```

Comment:
What are the expected rights?

On my side, I'd love that users can read but not create/update/delete.
Admin can do everything.